extensions/starttls-dialback.xml

```diff
@@ -285,16 +310,126 @@ example              example
   type='valid'/>
       ]]></example>
   </section2>
+  <section2 topic='Same Certificate shortcut' anchor='samecert'>
+    <p>If during the initial connection, home.montague.example is not able to determine that the certificate presented is trustworthy but the Authoritative Server presents the same certificate as the Originating Server, the &lt;db:verify/&gt; step can be elided.</p>
+    <p>Note: the Receiving Server MUST still check that the hostname in the certificate matches.</p>
+    <p>Essentially, this replaces the Dialback Key Validation step from &xep0185; with the somewhat more elaborate proof of posession of the private key associated with the certificate.</p>
+  <code><![CDATA[
+orchard.capulet.     home.montague.
+example              example 
+(as Initiating)      (as Receiving
+   Server)              Server)
+----------------     -------------
+    |                    |
+    |  [if necessary,    |
+    |   perform DNS      |
+    |   lookup on        |
+    |   Target Domain,   |
+    |   open TCP         |
+    |   connection,      |
+    |   and establish    |
+    |   stream]          |
+    | -----------------> |
+    |  (ID D60000229F)   |
+    |                    |
+    |      send          |               capulet.example
+    |  dialback key      |             (as Authoritative 
+    | -----(STEP 1)----> |                 Server)
+    |                    |             -----------------
+    |                    |  [if necessary,    |
+    |                    |   perform DNS      |
+    |                    |   lookup on        |
+    |                    |   Sender Domain,   |
+    |                    |   open TCP         |
+    |                    |   connection,      |
+    |                    |   and establish    |
+    |                    |   stream]          |
+    |                    | -----------------> |
+    |                    | [observe certificate
+    |                    |  correct for capulet.example
+    |                    |  as per RFC 6125 and same as
```

Comment:
"certificate correct as per RFC 6125" implies it has a valid trust path, doesn't it?